communities/views.py

```diff
@@ -1,3 +1,73 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views import View
+from django.http  import JsonResponse
+
+from .models         import Community
+from users.utils     import login_decorator
+
+class CommunityView(View):
+    def get(self, request, product_id):
+        communities   = Community.objects.filter(product_id=product_id)
+        # creator_id    = Creator.objects.get(id=product_id)
+        announcements = Community.objects.filter(product_id=product_id)
+
+        postings = [
+            {
+                'name': community.user.id,
+                'product_id': community.product.id,
+                'content': community.content,
+                'id' : community.id
+
+            } for community in communities if not community.parent_review.id
+        ]
+
+        comments = [
+            {
+                'user_id' : community.user.id,
+                'name' : community.user.name,
+                'product_id' : community.product.id,
+                'content': community.content,
+                'id': community.id,
+                'post_id': community.parent_review.id
+
+            } for community in communities if community.parent_review.id
+        ]
+        return JsonResponse({'postings':postings, 'comment':comments}, status=200)
+
+    @login_decorator
+    def post(self, request):
+        parent_review_id = request.POST.get('post_id', None)
+
+        data = json.loads(request.body)
+
+        Community.objects.create(
+            user_id = request.user,
+            content = data['content'],
+            product_id = data['product_id'],
+            parent_review_id = parent_review_id
+        )
+        
+        return JsonResponse({'message':'success'}, status=201)
```

Comment:
피드백 communities, reviews 분리

```suggestion
class CommunityView
    def post(self, request):
        # community = 게시글  POST :8000/community
        data = json.loads(request.body)

        Community.objects.create(
            user_id = request.user,
            content = data['content'],
            product_id = data['product_id']
        )
        
        return JsonResponse({'message':'success'}, status=201)
```
```python
class CommunityReviewView
    def post(self, request, commnity_id):
        # community = 게시글  POST :8000/community/1/reviews
        data = json.loads(request.body)

        Community.objects.create(
            user_id = request.user,
            content = data['content'],
            product_id = data['product_id']
            parent_id = community_id
        )
        
        return JsonResponse({'message':'success'}, status=201)

```